R/clean_data.R

```diff
@@ -94,7 +97,8 @@ clean_data <- function(data,
                                      subject_id_format   = NULL,
                                      prefix              = "PS",
                                      suffix              = "P2",
-                                     range               = c(1L, 100L))) {
+                                     range               = c(1L, 100L),
```

Comment:
Are the default values chosen based on anything or are they just arbitrary. If so what happens if you set them to NULL

Reply:
Good point - I will set the default values to `NULL`.